rust/worker/src/execution/orchestration/hnsw.rs

```diff
@@ -143,6 +146,8 @@ pub(crate) struct HnswQueryOrchestrator {
     result_channel: Option<
         tokio::sync::oneshot::Sender<Result<Vec<Vec<VectorQueryResult>>, Box<dyn ChromaError>>>,
     >,
+    // information from the frontend
+    collection_version: Option<i32>,
```

Comment:
should we also pass the log offset and use that to pull logs?

Reply:
We should do this. Need to address in a separate PR.